tools/proto2jsonschema/gen.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+set -e
+
+# Only run in repo root
+currentDir=${PWD##*/}
+
+if [ "$currentDir" != "google-cloudevents" ] ; then
+  echo "Error: Script must be run in google-cloudevents root.";
+  exit 1;
+fi
+
+# Install protoc
+case "$OSTYPE" in
+  linux*)
+    PROTOBUF_PLATFORM=linux-x86_64
+    PROTOC=tmp/bin/protoc
+    ;;
+  win* | msys* | cygwin*)
+    PROTOBUF_PLATFORM=win64
+    PROTOC=tmp/bin/protoc.exe
+    ;;
+  darwin*)
+    PROTOBUF_PLATFORM=osx-x86_64
+    PROTOC=tmp/bin/protoc
+    ;;
+  *)
+    echo "Unknown OSTYPE: $OSTYPE"
+    exit 1
+esac
+
+rm -rf tmp
+mkdir tmp
+cd tmp
+echo "- Downloading protobuf tools"
+PROTOBUF_VERSION=3.12.3
+curl -sSL \
+  https://github.com/protocolbuffers/protobuf/releases/download/v$PROTOBUF_VERSION/protoc-$PROTOBUF_VERSION-$PROTOBUF_PLATFORM.zip \
+  --output protobuf.zip
+unzip -q protobuf.zip
+cd ..
+
+echo "- Setting up protoc plugin (chrusty/protoc-gen-jsonschema)"
+go get -v github.com/chrusty/protoc-gen-jsonschema/cmd/protoc-gen-jsonschema
```

Comment:
If you wanted to pin a specific version of the protoc plugin, it would be on this line.